tilings/tiling.py

```diff
@@ -1154,6 +1154,11 @@ def partial_place_col(self, idx: int, direction: int) -> Tuple["Tiling", ...]:
     def extra_parameters(self) -> Tuple[str, ...]:
         return tuple("k_{}".format(i) for i in range(len(self._assumptions)))
 
+    def possible_parameters(self, n: int) -> Iterator[Dict[str, int]]:
+        parameters = [self.get_parameter(ass) for ass in self.assumptions]
+        for values in product(*[range(n + 1) for _ in parameters]):
```

Comment:
For this function to be correct with assumptions of greater length we need to be more clever here. I think at the moment it would be ok to simply raise an `NotImplementedError` if any assumption contains gridded perms of length >2

Reply:
Thats fair, I'll add it.